office/company_events.md

```diff
@@ -1,7 +1,16 @@
 # source{d} Events
 
 ## Get togethers
-Once a month we organise a get together where we either all go to a bar where free drinks will be served from 19:00-24:00 or we go for some group activities like salsa classes, karting or other fun stuff. Every sourcerer and their +1s are invited to join. Get togethers are the third Wednesday of the month. 
+Once a month we organise a get together where we either all go to a bar where free drinks will be served from 19:00-24:00 or we go for some group activities like salsa classes, karting or other fun stuff. That means, every Get together can be different and always depends on the activity.
+Some activities let us be more flexible (beers) while others have to follow a specific schedule (classes). We can generalize:
+  - The event is on the calendar from now to forever, it is the third Wednesday of the month.
+  - The starting time usually is around 7 pm and the duration is around 2-3 hours. 
```

Comment:
Some get together elapsed 1h (yoga class, for example); is it ok for next events?